src/components/App.tsx

```diff
@@ -2718,15 +2720,19 @@ class App extends React.Component<AppProps, AppState> {
         event,
         scenePointerX,
         scenePointerY,
-        this.state.editingLinearElement,
-        this.state.gridSize,
+        this.state,
       );
+      if (!editingLinearElement) {
+        return;
+      }
```

Comment:
this is annoying since it should never happen in practice (and if it does I'm not sure exiting is the right thing to do here), but it requires you to read the LinearEditor code to find out. IOW, it's not semantic.

Solutions are: 

- make `handlePointerMove`  appState param into `MarkNonNullable<AppState, "editingLinearElement">`, but it'd require either asserting in callsites, or using a type guard — which may not be that terrible: `if(isPropNonNullable(appState, "editingLinearElement"))`
- pass `editingLinearElement` as another arg
- do nothing...